src/swap/SwapReviewScreen.tsx

```diff
@@ -139,13 +142,10 @@ export function SwapReviewScreen() {
     ValoraAnalytics.track(SwapEvents.swap_review_submit, {
       toToken,
       fromToken,
-      // TODO: Add fee to total when enabled
-      usdTotal: +divideByWei(swapResponse.unvalidatedSwapTransaction.buyAmount).multipliedBy(
+      usdTotal: +divideByWei(swapResponse.unvalidatedSwapTransaction[swapAmountParam]).multipliedBy(
         swapResponse.unvalidatedSwapTransaction.price
       ),
-      fee: +divideByWei(swapResponse.unvalidatedSwapTransaction.sellAmount).multipliedBy(
-        swapFeeDecimal
-      ),
+      amount: swapAmount[updatedField] ?? '',
```

Comment:
i also think we should only do a buy OR sell amount here, because both will be available at this point. or if we really want, we can do buy AND sell. but avoiding a property that can be both without being able to know which it is, would be nice